### PR TITLE
Update UG to be consistent with v1.2

### LINKED ISSUE #71: Update UG to be consistant with v1.2

### PR DESCRIPTION
**Pull request by kinjalagarwal1810**
Not all the information is up to date with v1.3 for now

close #71

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,41 +1,114 @@
...
+* Manage orders and points: Seamlessly add orders and update points to reflect customer purchases and interactions.
+* Update membership tiers: Elevate members' tiers based on their accumulated points, unlocking new rewards and benefits aligned with each tier.
+
+### Understanding Membership Tiers
+One of the core features of SweetRewards is the Membership Tier System. This system rewards your loyal customers by categorizing them into different tiers based on the points they have accumulated through purchases at your bakery. Each tier offers different benefits, such as discounts, free items, or exclusive products. Here is a breakdown of the tiers:
+
+Bronze Tier (T1): This is the starting tier for all new members. Members need to accumulate at least 100 points to move to the next tier.
+Silver Tier (T2): Members who have accumulated 200 points are upgraded to the Silver Tier, which offers more benefits than the Bronze Tier.
+Gold Tier (T3): This is the highest tier, reserved for members who have collected 500 points. Gold Tier members receive the best benefits, underlining their importance to your bakery.
```
Comment (kokerinks) [inline docs/UserGuide.md:55]: I think for now can include Platinum tier first? 
```
    public static final String[][] TIERS = {
            {"PLATINUM", "1000"},
            {"GOLD", "500"},
            {"SILVER", "200"},
            {"BRONZE", "0"},
    };
```

If we change this at a later time then we'll update the UG accordingly ba

Comment (kokerinks) [reply docs/UserGuide.md:55]: Also I think we might not need the T1 T2 T3 stuff anym, like can just have BRONZE, SILVER etc. be the tier names

Comment (kokerinks) [reply docs/UserGuide.md:55]: ok nvm i get it this is for v1.2 LOL can js resolve first

File: docs/UserGuide.md [hunk 2]
```diff
@@ -245,22 +351,59 @@ _Details coming soon ..._
...
 
+2. GUI opens off-screen
+
+   * 🚫 Problem: When using multiple screens, SweetRewards opens off the visible screen. 
+   * ✅ Solution: Delete the preferences.json file found in the same folder as the jar file. Restart SweetRewards.
+
+### Operational Questions
+1. Adding Points to Wrong Member
+    * 🚫 Problem: Points were mistakenly added to the wrong member. 
+    * ✅ Solution: Use the edit command to adjust the points for both involved members. Subtract the incorrectly added points from the wrong member and add them to the correct one. For example, edit 2 p/-50 to remove points and edit 3 p/50 to add them where needed.
```
Comment (kokerinks) [inline docs/UserGuide.md:371]: I think the edit command wasn't implemented like this? I gave it a try with edit 1 p/-50 and it didn't work

Comment (kokerinks) [reply docs/UserGuide.md:371]: Yeah it just directly replaces the old points with the new ones

**Review by kokerinks — APPROVED** [review]
Our new UG looks damn good 🤩🤩 thanks for the effort, LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (kinjalagarwal1810) [reply docs/UserGuide.md:55]: Yup! Im still trying to update it acc to v1.3 as much as I can for now to make life easier later so I'll implement these changes as well. Thanks for pointing out!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,41 +1,114 @@
...
+# Introduction
+Welcome to the SweetRewards User Guide! This guide is your comprehensive manual to navigating and maximizing the SweetRewards application. SweetRewards is a dynamic desktop application designed for small bakery owners to manage their loyalty program memberships efficiently. By leveraging the robustness of a Command Line Interface (CLI) while retaining the intuitive nature of a Graphical User Interface (GUI), SweetRewards delivers an optimized experience for fast typers and GUI enthusiasts alike.
+
+## Target Audience
+This guide is intended for small to medium-sized bakery owners or staff responsible for managing customer relationships and loyalty programs. We assume that users have a basic understanding of command-line interfaces and desktop applications but do not require advanced technical skills. SweetRewards is crafted to simplify your membership tracking, offering a seamless integration into your daily operations.
+
+## Table of Contents
```
Comment (kab-dot) [inline docs/UserGuide.md:12]: Nice table of contents!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -69,148 +142,166 @@ Shows a message explaining how to access the help page.
...
 
+### Updating membership: `addmship`
 
-### Editing a person : `edit`
+Updates the membership tier of a member in the loyalty program.
 
-Edits an existing person in the address book.
+🌟 Motivate your customers to keep coming back by using addmship to upgrade their membership tier. Higher tiers can offer better rewards, creating a sense of achievement and encouraging more purchases.
+
+Format: `addmship n/MEMBER_NAME ms/MEMBERSHIP_TEIR`
```
Comment (kab-dot) [inline docs/UserGuide.md:222]: just a small spelling mistake should be "tier"

**Review by kab-dot — COMMENTED** [review]
LGTM! Thanks for updating it!

File: docs/UserGuide.md (hunk 2 above)
Comment (kinjalagarwal1810) [reply docs/UserGuide.md:371]: Thank you for pointing it out!

File: docs/UserGuide.md (hunk 4 above)
Comment (kinjalagarwal1810) [reply docs/UserGuide.md:222]: Thank you for pointing it out!

**Review by kab-dot — COMMENTED** [review]
LGTM!

**Pull request merged**